x/leveragelp/client/cli/tx_add_pool.go

```diff
@@ -0,0 +1,109 @@
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/elys-network/elys/x/leveragelp/types"
+	"github.com/spf13/cobra"
+)
+
+// Governance command
+func CmdAddPools() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-pools [pool.json]",
+		Short: "Update leveragelp pools",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			title, err := cmd.Flags().GetString(cli.FlagTitle)
+			if err != nil {
+				return err
+			}
+
+			summary, err := cmd.Flags().GetString(cli.FlagSummary)
+			if err != nil {
+				return err
+			}
+
+			metadata, err := cmd.Flags().GetString(cli.FlagMetadata)
+			if err != nil {
+				return err
+			}
+
+			signer := clientCtx.GetFromAddress()
+			if signer == nil {
+				return errors.New("signer address is missing")
+			}
+
+			pool, err := readPoolJSON(args[0])
+			if err != nil {
+				return err
+			}
+
+			govAddress := sdk.AccAddress(address.Module("gov"))
+			msg := types.NewMsgAddPools(
+				govAddress.String(),
+				pool,
+			)
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			depositStr, err := cmd.Flags().GetString(cli.FlagDeposit)
+			if err != nil {
+				return err
+			}
+
+			deposit, err := sdk.ParseCoinsNormalized(depositStr)
+			if err != nil {
+				return err
+			}
+
+			govMsg, err := v1.NewMsgSubmitProposal([]sdk.Msg{msg}, deposit, signer.String(), metadata, title, summary)
+			if err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), govMsg)
+		},
+	}
+	cmd.Flags().String(cli.FlagTitle, "", "title of proposal")
+	cmd.Flags().String(cli.FlagSummary, "", "summary of proposal")
+	cmd.Flags().String(cli.FlagMetadata, "", "metadata of proposal")
+	cmd.Flags().String(cli.FlagDeposit, "", "deposit of proposal")
+	_ = cmd.MarkFlagRequired(cli.FlagTitle)
+	_ = cmd.MarkFlagRequired(cli.FlagSummary)
+	_ = cmd.MarkFlagRequired(cli.FlagMetadata)
+	_ = cmd.MarkFlagRequired(cli.FlagDeposit)
+
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func readPoolJSON(filename string) (types.Pool, error) {
+	var pool types.Pool
+	bz, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return types.Pool{}, err
+	}
+	err = json.Unmarshal(bz, &pool)
+	if err != nil {
+		return types.Pool{}, err
```

Comment:
When creating new pool, I think we should use default values for lpAmount and health. In current case governance can set health or lpAmount to anything